### PR TITLE
Fix: Text breaking

### PR DESCRIPTION
**Pull request by frontned**

### CODE REVIEW

File: src/components/Text/Text.st.css [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 .root {
     -st-states: typography(enum(smallTitle, runningText, listText, largeTitle)), mobile;
+    word-break: break-word;
```
Comment (jonathanadler) [inline src/components/Text/Text.st.css]: I'm not sure this is the behavior we want.
In addition, this CSS rule is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

I'll check with our designers

**Comment by jonathanadler** [comment]
@alexkronenberg
This pull requests makes the text components' words to break in the middle of there isn't enough space.
It's this something we want?
You can see the specification here https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

**Comment by frontned** [comment]
What we are originally trying to fix with this pull request is this: 
image

**Comment by alexkronenberg** [comment]
> What we are originally trying to fix with this pull request is this:
> image

@jonathanadler @frontned shouldn't this case have an ellipsis?

**Comment by frontned** [comment]
@alexkronenberg if it would be the case then other titles for example 'Title Title Title Title Title Title Title' would also have and ellipsis, instead of now they break to another line.

**Comment by frontned** [comment]
@alexkronenberg To be more clear, here is what I mean:
image

**Comment by jonathanadler** [comment]
@alexkronenberg
The `Text` component doesn't have ellipsis.
I'm also not sure that we want that by default. For instance we wouldn't want it for running text, and such.
We can add an `ellipsis` prop perhaps, which can come in handy.
But this is also something that the TPAs can add themselves.

And regarding the case @frontned is talking about, what do you think should be their case?
Should they use ellipsis instead?

BTW, @frontned, thanks for the image, super helpful 🙏

**Comment by frontned** [comment]
With out UX - Paulius Pamiskis we decided that the word-break solution will fit our case, because if we add ellipsis then normal titles with spaces will suffer. It's a question now, should we impelement it on our side and close this pull request?

**Pull request closed without merging**